### PR TITLE
[risk=low][no ticket]Update a few vulnerable packages

### PR DESCRIPTION
**Pull request by jmthibault79**
Remediating some of the findings in this Snyk report: https://app.snyk.io/org/all-of-us-researcher-workbench/project/681ed9fe-4281-4244-bf0a-3f64ed414704/

Explicitly pin our Spring Boot version to ~1.5.3.RELEASE~ (some chose higher or arbitrary versions) **now 1.5.22.RELEASE**
Update vulnerable Jackson version to 2.9.9

### CODE REVIEW

**Comment by jmthibault79** [comment]
These changes brought in the vulnerable logback-classic 1.1.11, so this now includes an upgrade of that package to 1.2.3

File: api/build.gradle [hunk 1]
```diff
@@ -179,7 +177,7 @@ buildscript {    // Configuration for building
     }
   }
   dependencies {
-    classpath 'org.springframework.boot:spring-boot-gradle-plugin:1.5.3.RELEASE'
+    classpath 'org.springframework.boot:spring-boot-gradle-plugin:1.5.22.RELEASE'
```
Comment (jmthibault79) [inline api/build.gradle]: Latest patch-version of the 1.5 series, Aug 2019

File: api/build.gradle [hunk 2]
```diff
@@ -300,6 +298,10 @@ repositories {   // repositories for Jar's you access in your code
   jcenter()
 }
 
+project.ext.GAE_VERSION = '1.9.64'
+project.ext.SPRING_BOOT_VERSION = '1.5.22.RELEASE'
+project.ext.JACKSON_VERSION = '2.9.9'
```
Comment (jmthibault79) [inline api/build.gradle]: Latest patch-version of the 2.9 series, May 2019

File: api/build.gradle (hunk 1 above)
Comment (calbach) [inline api/build.gradle]: Can this use the SPRING_BOOT_VERSION variable?

Comment (jmthibault79) [reply api/build.gradle]: I was unable to get that to work.

Comment (jaycarlton) [reply api/build.gradle]: @jmthibault79 can you please file a low-priority ticket on getting that to work? I suspect judicious use of variables will be important to the dev tool cleanup campaign.

Comment (jmthibault79) [reply api/build.gradle]: Good news everyone.  I figured it out for #2564 .

**Pull request merged**